app/views/dashboard/autoyast.xml.erb

```diff
@@ -9,7 +9,8 @@
         <interpreter>shell</interpreter>
         <source><![CDATA[
         #!/bin/sh
-        echo "master: <%= @controller_node %>" > /etc/salt/minion.d/master.conf
+        echo "master: <%= @controller_node %>" > /etc/salt/minion.d/minion.conf
```

Comment:
About this filename... it was fine as `master.conf` IMHO like: this is where the master is... (contents are: `master: location`)